src/utils/searchHelper.ts

```diff
@@ -42,3 +41,11 @@ export function getSearchWith(
   // we return a string to use it inside links
   return newParams.toString();
 }
+
+export const createCenturiesParams = (centuries: string[], century: string) => {
+  return {
+    centuries: centuries.includes(century)
+      ? [...centuries.filter(c => c !== century)]
```

Comment:
```suggestion
      ? centuries.filter(c => c !== century)
```
it's redundant, the filter method doesn't mutate the array